### PR TITLE
feat: use starter-kitty for redirect validator

### PR DESCRIPTION
**Pull request by alexanderleegs**
See [thread](https://opengovproducts.slack.com/archives/CK9GGK6EP/p1722841650033399). This PR modifies our redirect url validation.

### CODE REVIEW

File: package.json
Comment (karrui) [inline package.json:1]: since this is a monorepo, i think the package should be installed in the correct package!

**Review by karrui — CHANGES_REQUESTED** [review]
@alexanderleegs can help change the directory of the installed package to studio?

File: package.json
Comment (alexanderleegs) [reply package.json:1]: whoops, fixed in https://github.com/opengovsg/isomer/pull/478/commits/05bb46285bb68e020a68255e8359d948cdfa9a7e

**Review by karrui — APPROVED** [review]
lgtm

**Pull request merged**